src/lib/crypto/kimchi_bindings/stubs/binding_generation/src/main.rs

```diff
@@ -343,6 +346,10 @@ fn generate_bindings(mut w: impl std::io::Write) {
         });
 
         decl_module!(w, env, "VerifierIndex", {
+            decl_module!(w, env, "Lookup", {
+                decl_type!(w, env, CamlLookupsUsed => "lookups_used");
+                decl_type!(w, env, CamlLookupVerifierIndex<T1> => "t");
```

Comment:
my PR introduces `promote`, which I think is going to be nice to see how these changes impact the generated bindings. If you want to add it here as well it wouldn't be a bad idea.